packages/shared/src/utils/constants/contract.ts

```diff
@@ -392,104 +422,129 @@ export const getSupportedContracts = (is_high_low?: boolean) =>
     ({
         ACCU: {
             button_name: localize('Buy'),
+            feature_flag: getTradeFeatureFlag(TRADE.ACCUMULATORS),
```

Comment:
Instead of adding `feature_flag` in each contract object. wouldn't it better just filtering by key?

eg:
const DISABLED_CONTRACTS = ['SHARKFIN'];

Reply:
Agreed, the requirements for now is to add feature flags for new features only!

Reply:
@yashim-deriv @matin-deriv Let me explain my approach first please.
This approach gives us configuration that will work for **any** new trade type added to this object in future because:
1. the feature flag is uniquely formatted, so that 'CALL' is not mixed up between higher/lower and rise/fall, and works well for both cases:
    - when we compare feature flags against trade categories (e.g. 'rise_fall') on the main trading page
    - when we compare feature flags against contract_types (e.g. 'CALL') in Contract Details page and 3 tabs of Reports page. This object is accessed from there to get the assigned feature flag based on the contract_type. 
2. the feature flag has prefix of 'trade_' in order to be hidden from Endpoint page for production because no feature flag will be coming from API for DTrader, we simply don't want to let users check/uncheck this feature flag in production.
3. 'CALL' & 'PUT'/ 'TURBOSLONG' & 'TURBOSSHORT'/etc. are united into 1 trade feature with 1 feature flag.
4. all we need to do when we work on Sharkfin is to add a feature_flag field.

**Question 1:** Do you mean that I don't need to make this flags applicable to **any** trade type that we would like to disable in future? I wanted to avoid checks for specific ones trade_sharkfin, trade_bla_bla because we might have a lot of new trade types in future, even at the same time, and it might be a hell to go to all this places I changed in this PR and add additional checks for trade_sharkfin for example.
**Question 2:** The reason why I added the feature flag field to this object is to give an example of usage based on existing configuration for trade types. And we don't know yet what Sharkfin contract types will be called. If you like, I can remove feature_flag fields from all existing trade types, and add a comment to this object saying: "// when adding Sharkfin, please also add feature_flag field to its config". What do you think?